content/en/Platform Deep Dive/Assets/risk-advisories.md

```diff
@@ -83,3 +83,14 @@ Select a vulnerability to view detailed information on the [National Vulnerabili
 {{% alert title="Note" color="primary" %}}
 Potential vulnerabilities on the **Risk Advisory** tab don’t belong to your pentests. These are potential risks based on the <a href="https://www.cve.org/" target="_blank">Common Vulnerabilities and Exposures (CVE)</a> standard that we show for your reference.
 {{% /alert %}}
+
+## Subscribe to Email Alerts
+
+You can subscribe to email alerts for **potential critical vulnerabilities** discovered in your assets. We only send alerts for Critical vulnerabilities, based on the CVSS severity levels.
```

Comment:
```suggestion
You can subscribe to email alerts for **potential critical vulnerabilities** discovered in your assets. We only send alerts to critical vulnerabilities according to the [CVSS score](https://nvd.nist.gov/vuln-metrics/cvss). That's all vulnerabilities that have a score of 9 or higher.
```

Reply:
@Ikaneng Should we also specify the rating version?

My suggestion:

We only send alerts for Critical vulnerabilities with a score of 9 or higher, according to the [NVD CVSS v3.0 ratings](https://nvd.nist.gov/vuln-metrics/cvss).

Reply:
Yes, your suggestion works for me.

Reply:
@Ikaneng Thank you, I merged this pull request.